### PR TITLE
fio_perf: fix TypeError of the raw disk

### PR DESCRIPTION
**Pull request by hellohellenmao**
change the fio cmd to fio options to match the current framework

Signed-off-by: Tingting Mao <timao@redhat.com>

ID: 2029259

### CODE REVIEW

**Comment by hellohellenmao** [comment]
@yanan-fu Could you please help to review? Thanks.

**Comment by yanan-fu** [comment]
This is really a regression issue with patch https://github.com/autotest/tp-qemu/pull/2994, after script be changed, not all the configuration be updated.

Please update case `virtio_fs_perf` together.

File: qemu/tests/cfg/fio_perf.cfg [hunk 1]
```diff
@@ -52,9 +52,12 @@
             vhost_nic1 =
             remove_image = no
             Linux:
-                fio_cmd = "i=`/bin/ls /dev/[vs]db` && fio --rw=%s --bs=%s --iodepth=%s --runtime=1m --direct=1 --filename=$i --name=job1 --ioengine=libaio --thread --group_reporting --numjobs=%s --time_based --output=/tmp/fio_result &> /dev/null"
+                virtio_blk:
+                    fio_options = "--rw=%s --bs=%s --iodepth=%s --runtime=1m --direct=1 --filename=/dev/vdb --name=job1 --ioengine=libaio --thread --group_reporting --numjobs=%s --time_based --output=/var/tmp/fio_result &> /dev/null"
+                virtio_scsi:
+                    fio_options = "--rw=%s --bs=%s --iodepth=%s --runtime=1m --direct=1 --filename=/dev/sdb --name=job1 --ioengine=libaio --thread --group_reporting --numjobs=%s --time_based --output=/var/tmp/fio_result &> /dev/null"
```
Comment (yanan-fu) [inline qemu/tests/cfg/fio_perf.cfg:58]: Well, `$i` be removed and use hardcode `sdb/vdb`,  i am not very confident about the stability of this change, the diskname is fixed for all situation ?  @xuwei Could you help review ? Thanks!

**Comment by hellohellenmao** [comment]
> This is really a regression issue with patch #2994, after script be changed, not all the configuration be updated.
> 
> Please update case `virtio_fs_perf` together.

@xiagao Could you please help to handle this for it's ### virtio_fs_perf issues.

**Comment by xiagao** [comment]
> 2029259

OK, I will have a check.

**Comment by yanan-fu** [comment]
> > This is really a regression issue with patch #2994, after script be changed, not all the configuration be updated.
> > Please update case `virtio_fs_perf` together.
> 
> @xiagao Could you please help to handle this for it's `virtio_fs_perf` issues.

@hellohellenmao  IMO, it is nothing to do with virtio_fs,  just a param name change as the change with pre patch https://github.com/autotest/tp-qemu/pull/2994

A simple correction is enough.

**Comment by hellohellenmao** [comment]
> > > This is really a regression issue with patch #2994, after script be changed, not all the configuration be updated.
> > > Please update case `virtio_fs_perf` together.
> > 
> > 
> > @xiagao Could you please help to handle this for it's `virtio_fs_perf` issues.
> 
> @hellohellenmao IMO, it is nothing to do with virtio_fs, just a param name change as the change with pre patch #2994
> 
> A simple correction is enough.

@yanan-fu Xiagao is the feature own of virio_fs performance now. It's also okay for me to fix the `virtio_fs_perf`.

File: qemu/tests/cfg/fio_perf.cfg (hunk 1 above)
Comment (XueqiangWei) [reply qemu/tests/cfg/fio_perf.cfg:58]: I think it's OK. Use virtio_blk and virtio_scsi to distinguish the diskname vdb/sdb.
And tingting's codes, just update case \"single_disk.raw_disk.NVMe\" related options, LGTM

**Comment by xiagao** [comment]
> > > > This is really a regression issue with patch #2994, after script be changed, not all the configuration be updated.
> > > > Please update case `virtio_fs_perf` together.
> > > 
> > > 
> > > @xiagao Could you please help to handle this for it's `virtio_fs_perf` issues.
> > 
> > 
> > @hellohellenmao IMO, it is nothing to do with virtio_fs, just a param name change as the change with pre patch #2994
> > A simple correction is enough.
> 
> @yanan-fu Xiagao is the feature own of virio_fs performance now. It's also okay for me to fix the `virtio_fs_perf`.

I checked the previous pr, for virtiofs part, only need to change 
fio_cmd = "fio --rw=%s 
with
fio_options = '--rw=%s

@hellohellenmao Could you help update it?

**Comment by hellohellenmao** [comment]
@xiagao Updated. Could you please help to review? Thanks.

**Comment by xiagao** [comment]
> @xiagao Updated. Could you please help to review? Thanks.

LGTM for virtiofs part.

**Comment by yanan-fu** [comment]
@xiagao  Could you provide the test result with `virtio_fs_perf` ? Thanks!

**Comment by xiagao** [comment]
(1/1) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.fio_perf.shared_fs.virtio_fs_perf.auto.q35: PASS (108442.07 s)

**Review by yanan-fu — COMMENTED** [review]
Ack

**Pull request merged**